### DISCUSSION TITLE
[Feature Request]: need a new api to get torretn file

### DISCUSSION THREAD

**Discussion by trim21** [transmission/transmission#6817] [Ideas]
Hi, can we add a new api to retrieve added torrent file from transmission?

**Comment by killemov** [comment]
What's there to discuss?

**Comment by killemov** [comment]
I assume you mean using the web UI. If you need to have this functionality now you need to make an access path in your web server or make a symbolic link to Transmission's'  torrents directory in Transmission's' web folder. And then use [Shift](https://github.com/killemov/Shift) and configure the `torrentLink` and `torrentLinkEnabled` settings.

**Reply by trim21** [reply]
no, I mean new native rpc api, not workaround

**Reply by killemov** [reply]
> If you need to have this functionality now
